nemo/core/classes/modelPT.py

```diff
@@ -805,13 +805,31 @@ def configure_optimizers(self):
         else:
             return [self._optimizer], [self._scheduler]
 
+    def propagate_model_guid(self):
```

Comment:
No need to cache the propogation, do it every time its called. Its relatively cheap and avoids surprises in the future.

Reply:
sure, just removed the cache